frontend/src/Components/UserProfileInvites.tsx

```diff
@@ -227,11 +233,11 @@ const Invite = (props: {
         {!!invite.reason && <div className={classNames(commentStyles.content, styles.content)}><ContentComponent content={invite.reason}/></div>}
 
         {props.active && !!invite.code && <>
-            <div className="code"><Link to={`//${process.env.REACT_APP_ROOT_DOMAIN}/invite/${invite.code}`}
+            <div className={classNames('code', (idx === regeneratedIdx ? 'regenerated' : ''))}><Link to={`//${process.env.REACT_APP_ROOT_DOMAIN}/invite/${invite.code}`}
```

Comment:
nit: [`classNames`](https://www.npmjs.com/package/classnames#usage) support syntactic sugar for conditional classes:
```js
classNames('code', {'regenerated': idx === regeneratedIdx})
```